### DISCUSSION TITLE
Referencing a bookmark with its associated page number

### DISCUSSION THREAD

**Discussion by EmileRolley** [dolanmiu/docx#2223] [Q&A] [answered]
Hi,

I'm looking to find a way to be able to get the page number of a section from it bookmark's id. Is it possible to do this?

Thanks <3

**Comment by dolanmiu** [comment]
Not possible I am afraid.
Page numbers is something that is generated by Microsoft Word etc at runtime

**Reply by EmileRolley** [reply, marked as answer]
In fact, it's possible (and pretty easy) with [page references](https://docx.js.org/#/usage/hyperlinks?id=internal) @dolanmiu :)

**Reply by dolanmiu** [reply]
Thanks for this! 

Sorry i misunderstood the question
